### DISCUSSION TITLE
Hide URL of a Link

### DISCUSSION THREAD

**Discussion by ldmax** [nvim-neorg/neorg#935] [Q&A] [answered]
Dear community,

From help document, I know the format of a link is like below:\
```norg
{https://pit-ray.github.io/win-vind}[win-vind]
```
Is it possible to hide the URL (https://pit-ray.github.io/win-vind), leaving only the `win-vind` but the `win-vind` is click-able (see below example)? Thanks in advance!

```norg
win-vind
```

**Comment by inhake2** [comment]
I’m not sure to understand what you mean (sorry if I’m wrong !)
By "hide", do you mean conceal ? It should be the case, concealed links are "clickable" with return.
Or do you want `concealcursor=n` (or `+=n`) ?

**Reply by ldmax** [reply]
hey snomics, thanks for your answer!

**Comment by SteveWolligandt** [comment, marked as answer]
You need to do
`:set conceallevel=1`.

However if you have long links you get empty lines:
![image](https://github.com/nvim-neorg/neorg/assets/5815171/90789a99-e227-4022-ab90-faf7e1a7afeb)
![image](https://github.com/nvim-neorg/neorg/assets/5815171/3c93f236-6387-462c-aaef-0662a57d8f34)

Is this expected behavior?

**Reply by ldmax** [reply]
Thanks dude!
